server.py

```diff
@@ -574,5 +574,6 @@ def addRestaurantReview(restaurant_id):
     review["restaurant_id"] = restaurant_id
     reviews.append(review)
     return 'OK'
```

Comment:
I'm not sure how the original interface was, but it would be nice to see the ID of the review on the front end side. I believe the correct response would be the ID, not 'ok'

Reply:
Why would you need the ID on the front side?

Reply:
If our front end is caching reviews, it could be useful, depending on how that caching is built. I don't know/remember what your front end does, but in either case, if this is what the other backend did, that is fine.